lib/linter-views.coffee

```diff
@@ -110,9 +124,32 @@ class LinterViews
       )
       throw null
 
+  updateCurrentLine: (line) ->
+    if @_currentLine isnt line
+      @_currentLine = line
+      @_updateLineMessages()
+
+
+  _updateLineMessages: ->
```

Comment:
You are currently collecting messages from eachLinterEditor, I had suggested that we collect only active's.